cassandra_nodetool/datadog_checks/cassandra_nodetool/data/conf.yaml.example

```diff
@@ -1,32 +1,41 @@
+## All options defined here are available to all instances.
+#
 init_config:
 
-  ## @param nodetool - string - required
-  ## command or path to nodetool (e.g. /usr/bin/nodetool or docker exec container nodetool)
-  ## can be overwritten on an instance
-  ## Note: Agent v6.11+ on Windows runs as an unprivileged user (`ddagentuser`). That user needs to be granted
-  ## access to the nodetool installation directory for the check to work.  
-  ## The nodetool installation also sets some environment variables (e.g. `CASSANDRA_HOME` and `DSCINSTALLDIR`),
-  ## but sets them as variables only for the user doing the nodetool installation. Those environment variables should
-  ## be set as system-wide variables.
-  #
-  # nodetool: /usr/bin/nodetool
+    ## @param nodetool - string - required
+    ## command or path to nodetool (e.g. /usr/bin/nodetool or docker exec container nodetool)
+    ## can be overwritten on an instance
+    ## Note: Agent v6.11+ on Windows runs as an unprivileged user (`ddagentuser`). That user needs to be granted
+    ## access to the nodetool installation directory for the check to work.
+    ## The nodetool installation also sets some environment variables (e.g. `CASSANDRA_HOME` and `DSCINSTALLDIR`),
+    ## but sets them as variables only for the user doing the nodetool installation. Those environment variables should
+    ## be set as system-wide variables.
+    #
+    nodetool: nodetool
+
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
```

Comment:
```suggestion
    ## Attaches the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
```

Reply:
This is from a base template. This file is generated